src/rpc/core_rpc_server.cpp

```
@@ -3652,10 +3652,8 @@ namespace cryptonote::rpc {
     // ---------------------------------------------------------------------------------------------
     if (req.encrypted_value.size() % 2 != 0)
       throw rpc_error{ERROR_INVALID_VALUE_LENGTH, "Value length not divisible by 2, length=" + std::to_string(req.encrypted_value.size())};
-
-    if ((req.encrypted_value.size() >= (bns::mapping_value::BUFFER_SIZE * 2)) && !(req.type =="wallet"))
-      throw rpc_error{ERROR_INVALID_VALUE_LENGTH, "Value too long to decrypt=" + req.encrypted_value};
-
+    if (req.encrypted_value.size() > bns::mapping_value::BUFFER_SIZE * 2)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)